index.html

```diff
@@ -60,125 +65,166 @@ <h2 class="header">Tonic</h2>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
-                    <button class="project-button">See Project</button>
+                    <button type="submit" class="project-button">See Project</button>
+                    </div>
                 </div>
             
-            <div class="work">
-                <div class="work-image">
+            <div class="work reverse">
+                <div class="work-image lost">
```

Comment:
- Please adjust the images on the image below, so that it looks like the other image set up in your design Because the particular image is not aligned properly as the screen size is resized. As it has lots of spaces on the left, but your other project images are ok, just work on this one.

##### The image
![Screenshot from 2022-06-04 08-39-07](https://user-images.githubusercontent.com/58771507/171989809-84e5e633-6163-4213-b767-4ffc3420bce9.png)